ml_project/features/abs_transformer.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class AbsTransformer(BaseEstimator, TransformerMixin):
+    def fit(self, X, y=None):
+        return self
+
+    def transform(self, X, y=None):
```

Comment:
make more complex example for fit and transform
during transform check that transformer was fitted